index.html

```diff
@@ -87,108 +87,49 @@ <h1 class="work">
         <div class="indicator"></div>
 
     </section>
-    <section class="grid-container">
-        <div class="pic-holder-container">
-            <div class="holder-container">
-                <h3>Multi-Post Stories <span>Gain + Glory</span></h3>
-                <ul class="stories">
-                    <li> Ruby on rails</li>
-                    <li> CSS</li>
-                    <li>JavaScript</li>
-                    <li>HTML</li>
-                </ul>
-                <div>
-                    <button class="get-button">
-                        See Project
-                    </button>
-                </div>
-            </div>
-        </div>
+    <div id="carddetailsmain">
+        <div id="carddetails">
+            <button class="dclosebtn"><img src="./close.svg" alt="" width="30px"></button>
+            <img src="./capture.svg" alt="" class="capture">
 
-        <div class="pic-holder-container">
-            <div class="holder-container">
-                <h3>Multi-Post Stories <span>Gain + Glory</span></h3>
-                <ul class="stories">
-                    <li> Ruby on rails</li>
-                    <li> CSS</li>
-                    <li>JavaScript</li>
-                    <li>HTML</li>
-                </ul>
-                <div>
-                    <button class="get-button">
-                        See Project
-                    </button>
+            <div class="flex spaceb">
+                <h2 id="tit">
+                    Keeping track of hundreds of <br>
+                    components
+                </h2>
+                <div class="flexBtn">
+                    <a href="https://wils-fourr-speed.github.io/My-portfolio/" class="dbtn">see live <img src="live.svg" alt=""></a>
+                    <a href="https://github.com/Wils-FOURR-speed/My-portfolio" class="dbtn">see source <img src="github.svg" alt=""></a>
```

Comment:
- [ ] As requested by the previous reviewer kindly consider making sure that these buttons are also visible on your mobile pop-up to make sure that your website is interactive you can do so by adding a scroll on the pop-up so if the content is more there will be a way to scroll up and down in the popup
<img width="737" alt="Screen Shot 2023-06-24 at 23 34 38" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/60146030/a94db237-9cc8-458c-b82b-c11792f17e28">